includes/class-sensei-lesson.php

```diff
@@ -4475,6 +4483,32 @@ public function log_initial_publish_event( $lesson ) {
 		sensei_log_event( 'lesson_publish', $event_properties );
 	}
 
+	/**
+	 * Check if a user is the lesson author.
+	 *
+	 * @since 3.0.2
+	 *
+	 * @param int $lesson_id ID of lesson being checked.
+	 * @param int $user_id ID of user being checked. Defaults to current user ID.
+	 * @return boolean Returns TRUE if user is the lesson author, returns FALSE otherwise.
+	 */
+	private static function is_lesson_author( $lesson_id, $user_id = null ) {
```

Comment:
I don't think this needs to be `static`.

Reply:
Done!